src/DetailsView/components/save-assessment-button.tsx

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { InsightsCommandButton } from 'common/components/controls/insights-command-button';
```

Comment:
Lots of these imports are unused. The eslint extension should dim the unused imports and on mouse-over offer a Quick Fix to remove them.